src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -9480,6 +9480,39 @@ public void MissingCompilerAssembly()
                 result.Output.Trim());
         }
 
+        [WorkItem(406649, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=484417")]
+        [ConditionalFact(typeof(WindowsOnly))]
+        public void MicrosoftDiaSymReaderNativeAltLoadPath()
+        {
+            var dir = Temp.CreateDirectory();
+            var cscDir = Path.GetDirectoryName(s_CSharpCompilerExecutable);
+            
+            // copy csc and dependencies except for DSRN:
+            foreach (var filePath in Directory.EnumerateFiles(cscDir))
+            {
+                var fileName = Path.GetFileName(filePath);
+
+                if (fileName.StartsWith("csc") ||
+                    fileName.StartsWith("System.") ||
+                    fileName.StartsWith("Microsoft.") && !fileName.StartsWith("Microsoft.DiaSymReader.Native"))
+                {
+                    dir.CopyFile(filePath);
+                }
+            }
+
+            dir.CreateFile("Source.cs").WriteAllText("class C { void F() { } }");
+
+            var cscCopy = Path.Combine(dir.Path, "csc.exe");
+
+            var result = ProcessUtilities.Run(
+                cscCopy, 
+                arguments: "/nologo /t:library /debug:full /deterministic Source.cs",
+                workingDirectory: dir.Path, 
+                additionalEnvironmentVars: new[] { KeyValuePair.Create("MICROSOFT_DIASYMREADER_NATIVE_ALT_LOAD_PATH", cscDir) });
```

Comment:
>KeyValuePair.Create("MICROSOFT_DIASYMREADER_NATIVE_ALT_LOAD_PATH", cscDir) }); [](start = 51, length = 78)

It feels like we should have tests for cases when the variable isn't there, or has null, or an empty string as its value.

Reply:
We have a lot of test cases for when it's not defined - all the other PDB tests. Null and empty values are the same as the variable not being set. I don't think we need extra tests for those.